bpf/README.md

```diff
@@ -16,17 +16,35 @@ This document will help you get started.
 We support BPFs only on ARM64 and x86_64 architectures.
 
 1. VM with Ubuntu 20.04+ (or any other Linux distro with the kernel 5.8+ - Docker doesn't work for many BPF features)
-2. Install `clang` (required for compiling BPF code) - `apt install clang` should be enough
+2. Install `clang` and `llvm` (required for compiling BPF code) - `apt install clang llvm` should be enough
 
 You can also use our Dockerfile to build the environment:
 
 ```bash
 make -C build.assets bpf-bytecode
 ```
 
+You can build BPF files faster using your local clang compiler using `make bpf-bytecode`.
+You will still need to run `make -C build.assets bpf-bytecode` to create reproducible files,
+otherwise the CI job that runs `make bpf-up-to-date` will fail.
+
 Note that there have been some issues with command tracing reading commands and arguments in a arm64 VM. You may
 need to test/develop on an EC2 instance instead.
 
+## Testing
```

Comment:
Any instructions for running on macOS?

Reply:
I'm curious how you are running it. I do have a lima setup, that is probably what I would try to use if needed.

Reply:
I mostly develop on a desktop tbh, when I need to I use the previously issued Thinkpad as well. A lot easier to dev on Linux hardware rather than in an EC2 instance